### ISSUE TITLE
Define variables with underscore

### ISSUE THREAD

**Issue by swopnil123** [ronkok/Hurmet#20]
Is it possible to to define variables with underscore or multiple subscripts? Like this

![image](https://github.com/ronkok/Hurmet/assets/17706904/e14d5eb1-0794-4be9-bfc7-0ca684316562)
or 

![image](https://github.com/ronkok/Hurmet/assets/17706904/a9e3aa4d-b80f-4bf3-aff8-6ea46b01cdc7)

**Comment by ronkok** [comment]
No. An underscore character is taken to mean the beginning of a subscript. I can't make an underscore mean two things at once.
